dbs/migrations/20240626210951-create-menu.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
         allowNull: false,
       },
       price: {
-        type: Sequelize.STRING,
+        type: Sequelize.INTEGER,
```

Comment:
@codedCoderr I'm not saying it's wrong, I'm only trying to understand the reasons behind the decision.